src/Monolog/Logger.php

```diff
@@ -311,9 +311,16 @@ public function addRecord($level, $message, array $context = array())
             static::$timezone = new \DateTimeZone(date_default_timezone_get() ?: 'UTC');
         }
 
+		$timestamp = microtime(true);
```

Comment:
this code should be inside the `if ($this->microsecondTimestamps && PHP_VERSION_ID < 70100) {`, to avoid doing useless work on newer PHP versions

Reply:
- @stof Can I kindly ask you for another check? :) 
- What should be the next step. Should it be merged to one commit or something like that? :)